it-tests/ContextualMenuOpen.test.ts

```diff
@@ -0,0 +1,52 @@
+import { ActivityBar, ContextMenu, EditorView,  SideBarView,  TextEditor, ViewControl,  ViewSection,  VSBrowser, WebDriver } from 'vscode-extension-tester';
+import { assert, expect } from 'chai';
+import * as path from 'path';
+import { checkEmptyCanvasLoaded, switchToKaotoFrame } from './Util';
+
+describe('Contextual menu opening', function () {
+	this.timeout(50000);
+
+	const workspaceFolder = path.join(__dirname, '../test Fixture with speci@l chars');
+
+	let driver: WebDriver;
+
+	before(async function() {
+		this.timeout(20000);
+		driver = VSBrowser.instance.driver;
+		await VSBrowser.instance.openResources(workspaceFolder);
+	});
+
+	afterEach(async function() {
+		const editorView = new EditorView();
+		await editorView.closeAllEditors();
+	});
+
+	it('Open Camel file with name my.yaml with right-click and check Kaoto UI is loading', async function () {
+        const control: ViewControl | undefined = await new ActivityBar().getViewControl('Explorer');
+        if(control === undefined) {
+            assert.fail('Not found the Explorer view');
+        }
+        const explorerView: SideBarView = await control.openView();
+        const workspaceSection: ViewSection = await explorerView.getContent().getSection('test Fixture with speci@l chars');
+        await workspaceSection.expand();
+        const myYamlItem = await workspaceSection.findItem('my.yaml');
+        if(myYamlItem === undefined) {
+            assert.fail('Cannot find the my.yaml file in explorer.')
+        }
+        const contextMenu: ContextMenu = await myYamlItem.openContextMenu();
```

Comment:
this is something which is not supported on vscode-extension-tester side currently for macOS. So you have to skip this test for `process.platform === 'darwin'` or you would have to workaround the openContext item `Open with Kaoto Graphical Editor for Camel` with some command palette equivalent for macOS

Reply:
the command i snot available on Palette for now on purpose. So I will have to skip on MacOS then